.github/workflows/identify-codeowners.yml

```
@@ -0,0 +1,19 @@
+name: Identify Codeowners
+
+on:
+  pull_request_target:
+    types: [opened, synchronize, reopened, ready_for_review]
+
+jobs:
+  identify-codeowners:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout and setup environment
+        uses: metamask/github-tools/.github/actions/checkout-and-setup@1299bb1de0c6974ae6d0a32c7e8897fe168239ac
+        with:
+          is-high-risk-environment: false
```

The `skip-allow-scripts` option didn't exist before, but now we want to turn it on everywhere in `metamask-extension`
```suggestion
          is-high-risk-environment: false
          skip-allow-scripts: true
```

---

is it available to use? I get an invalid action error when trying to add it.

---

Oh whoops sorry, the uses changed
```
      - name: Checkout and setup environment
        uses: MetaMask/action-checkout-and-setup@v1
        with:
          is-high-risk-environment: false
          skip-allow-scripts: true
```